src/wallet/feebumper.cpp

```diff
@@ -194,7 +194,7 @@ Result CreateTransaction(const CWallet* wallet, const uint256& txid, const CCoin
 
     // If the output would become dust, discard it (converting the dust to fee)
     poutput->nValue -= nDelta;
-    if (poutput->nValue <= GetDustThreshold(*poutput, ::dustRelayFee)) {
+    if (poutput->nValue <= GetDustThreshold(*poutput, GetDiscardRate(::feeEstimator))) {
```

Comment:
micro-nit: it looks like we use `IsDust()` in most(?) places in the code, and use GetDustThreshold() directly in just a couple.  Perhaps we should use `IsDust()` here instead (which apparently uses `<` rather than `<=`, so this would help us be consistent).